### DISCUSSION TITLE
Bluetooth change "getting stranded" risk?

### DISCUSSION THREAD

**Discussion by teslahorseman** [Kemmey/Kemmey-TeslaWatch-Public#562] [Q&A]
With the bluetooth key capability, does that mean the risk of getting "stranded" when your watch and/or car don't have internet connectivity has been mitigated? I'm asking because reviewers seem to imply that you don't need the internet connection anymore, yet your website/app description still cautions not being without a backup key because of the "getting stranded because of no internet" risk. What can you tell me?

Thanks!

Alex

**Comment by Kemmey** [comment]
Hey Alex,

The risk has definitely been reduced dramatically. I always recommend having your keycard with you as a backup - if you're going without, be prepared to have a backup solution in an acceptable timeframe! Ie - don't go places where you can't afford to wait for someone to come help you out in case you get locked out! In practice, it's been rock-solid for me since December...

**Comment by teslahorseman** [comment]
Thank you for the reply! It's good to know that an internet connection is no longer required for authentication to work. I understand that there can be "glitches" here and there, and I always do carry my key card in my wallet anyway, "just in case". I don't even trust the Tesla app 100% of the time.
